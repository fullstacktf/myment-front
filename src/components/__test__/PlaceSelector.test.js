import SelectPlace from '../PlaceSelector.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';

describe('SelectPlace', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(SelectPlace, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  }),
    test('Check props', () => {}),
    test('check Contents props', () => {});
});
