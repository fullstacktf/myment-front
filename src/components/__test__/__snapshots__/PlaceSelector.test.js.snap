// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectPlace render component 1`] = `"<div class=\\"columns\\"><div class=\\"column is-4\\"><div class=\\"select dropdown\\"><select><option value=\\"\\">Select a Country</option> <option value=\\"España\\">España</option></select></div></div> <div class=\\"column is-4\\"><div class=\\"select dropdown\\"><select disabled=\\"disabled\\"><option value=\\"\\">Select a City</option> </select></div></div> <div class=\\"column is-4\\"><div class=\\"select dropdown\\"><select disabled=\\"disabled\\"><option value=\\"\\">Select a Zone</option> </select></div></div></div>"`;
